tests/unit/plugins/modules/net_tools/test_dnsimple_info.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+from ansible_collections.community.general.plugins.modules.net_tools import dnsimple_info
+from ansible_collections.community.general.tests.unit.plugins.modules.utils import AnsibleFailJson, ModuleTestCase, set_module_args
+from ansible_collections.community.general.tests.unit.compat.mock import patch
+import pytest
+import sys
+
+
+dnsimple = pytest.importorskip('dnsimple')
+mandatory_py_version = pytest.mark.skipif(
+    sys.version_info < (3, 6),
+    reason='The dnsimple dependency requires python3.6 or higher'
+)
+
+
+class TestDNSimple(ModuleTestCase):
+    """Main class for testing dnsimple module."""
+
+    def setUp(self):
+        """Setup."""
+        super(TestDNSimple, self).setUp()
+        self.module = dnsimple_info
+
+    def tearDown(self):
+        """Teardown."""
+        super(TestDNSimple, self).tearDown()
+
+    def test_without_required_parameters(self):
+        """Failure must occurs when all parameters are missing"""
+        with self.assertRaises(AnsibleFailJson):
+            set_module_args({})
+            self.module.main()
```

Comment:
You probabl ywant to add some more interesting tests, like one where `name` and `record` are not specified, one where `name` is specified but record is not, and one where both are specified.